components/develop/modules/ROOT/pages/sfuel/index.adoc

```diff
@@ -0,0 +1,36 @@
+= sFUEL (Gas Token)
+:page-aliases: sfuel-gas-token.adoc
+
+SKALE Fuel or **sFUEL** is the native gas token of SKALE Chains. This is similar -- but different! -- to Ether on the Ethereum blockchain. To interact with a SKALE chain, sFUEL tokens are required to conduct transactions. Unlike Ether and other gas tokens, sFUEL has no economic value: it's a **FREE** resouse making transactions on SKALE 100% free all the time.
+
+[NOTE]
+Users must have sFUEL in order to transact through their wallets. Alternativley, transactions can be "paid for" by utilizing xref:../proof-of-work.adoc[Proof of Work].
```

Comment:
paid for --> mined

Reply:
Resolved by c55f276a66bff1fc74f46655df91f34f7893acac